### PR TITLE
complete css-cascade

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

### CODE REVIEW

File: css-cascade/specificity.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  color: green;
+}
+
+#timbl {
+  color: red;
+}
+
+#verou {
+  color: black !important;
+}
```
Comment (BrettAlbright) [inline css-cascade/specificity.css]: these selectors should be in reverse order to demonstrate the power of specificity.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by K-Karneboge** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**